js/dataframe/shared/Table.svelte

```diff
@@ -325,7 +325,6 @@
 
 	async function handle_cell_click(i: number, j: number): Promise<void> {
 		if (dequal(editing, [i, j])) return;
-		if (dequal(selected, [i, j])) return;
```

Comment:
Not sure why we needed this line, but it was causing the "double select" bug. cc @pngwn

Reply:
Hmm, okay I guess that makes sense. Could introduce other bugs thought because we refocus at the bottom. Might be okay though.